### DISCUSSION TITLE
What is the difference between datatracker.ietf.org & www.rfc-editor.org and which is Official URL for RFC ?

### DISCUSSION THREAD

**Discussion by Jxck** [ietf-tools/datatracker#7259] [Help / Questions]
Some RFC has multiple URL.

- https://tools.ietf.org/html/rfc9110
- https://datatracker.ietf.org/doc/html/rfc9110
- https://www.rfc-editor.org/rfc/rfc9110.html

IIUC, tools.ietf.org is deprecated and redirect to datatracker.
datatracker has some metadata for RFC and rfc-editor looks focusing on reading content itself.
But I don't know what is the purpose to have multiple site and how they utilize.

Q: if I put link into some official document (like Paper, Article, Manual, Book etc).
Which is the best URL to link RFC ?

**Comment by Jxck** [comment]
It'd be helpful to have official guidance from IETF for here.
If I missed it, please let me know.

**Reply by kesara** [reply]
I hope these resources might help you:
 * https://authors.ietf.org/en/references-in-rfcxml
 * https://www.rfc-editor.org/styleguide/part2/

**Comment by cabo** [comment]
If you look at rfcs citing other rfcs, these all use rfc-editor.org. It couldn’t be more official. Sent from mobile, sorry for terseOn 26. Mar 2024, at 14:09, Jxck ***@***.***> wrote:﻿
It'd be helpful to have official guidance from IETF for here.
If I missed it, please let me know.

**Reply by Pham1993** [reply]
[]()

**Comment by rjsparks** [comment]
You should point to rfc-editor.org

However, be aware that the current style guide says to point to
https://www.rfc-editor.org/info/rfc9110 

There is ongoing discussion about making the page referenced contain both content and metadata.

**Comment by Jxck** [comment]
> You should point to rfc-editor.org

Got it. Thanks.

> However, be aware that the current style guide says to point to
> https://www.rfc-editor.org/info/rfc9110

Hmm, it seems hard choice for me...🤔

**Comment by Pham1993** [comment]
https://github.com/ietf-tools/datatracker/discussions/7259

**Comment by ghost** [comment]
> Some RFC has multiple URL.
> 
> 
> 
> - https://tools.ietf.org/html/rfc9110
> 
> - https://datatracker.ietf.org/doc/html/rfc9110
> 
> - https://www.rfc-editor.org/rfc/rfc9110.html
> 
> 
> 
> IIUC, tools.ietf.org is deprecated and redirect to datatracker.
> 
> datatracker has some metadata for RFC and rfc-editor looks focusing on reading content itself.
> 
> But I don't know what is the purpose to have multiple site and how they utilize.
> 
> 
> 
> Q: if I put link into some official document (like Paper, Article, Manual, Book etc).
> 
> Which is the best URL to link RFC ?
